packages/api/src/libs/utils.ts

```diff
@@ -148,3 +149,30 @@ export function parseEnvArray(str: string, delimiter = ","): string[] {
   if (!str.includes(delimiter)) return [];
   return str.split(delimiter).map((x) => x.trim());
 }
+
+export function getWeb3Websocket(url: string, options: Obj = {}) {
+  // pulled from common/src/ProviderUtils.ts
```

Comment:
i considered importing the getWeb3, but from what i could tell, there is other logic for retrying calls which i wasnt sure i needed https://github.com/UMAprotocol/protocol/blob/master/packages/common/src/ProviderUtils.ts#L37

Reply:
The default retry is a passthrough (no retries), but you can change some env variables and provide a retry config that will just make failing requests retry. Not super opinionated either way.

Reply:
oh interesting, good to know